src/modules/feature-modules/admin/pages/service-users/service-users-find.component.ts

```diff
@@ -14,13 +16,35 @@ export class PageServiceUsersFindComponent extends CoreComponent implements OnIn
     { type: 'button', label: 'New user', url: '/admin/service-users/new' }
   ];
 
-  constructor() {
+  form = new FormGroup({
+    search: new FormControl()
+  });
 
-    super();
-    this.setPageTitle('Service users');
+  serviceUsers?: searchUserEndpointDTO[];
```

Comment:
Because you are initializing it with an empty array, this don't need to be optional. 

Also, I recommend that you rename this variable, like usersList or serviceUsersList, so it don't gets confused with serviceUsersService...